11_Exercise/11_Excersice.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>11_Exercise</title>
+  <link rel="stylesheet" href="css/app.css">
+</head>
+<body>
+  <div class="container">
+    <input id="myBtn" value=""><br>
+    <button onclick="button1()">1</button>
+    <button onclick="button2()">2</button>
+    <button onclick="button3()">3</button>
+    <button onclick="buttonSum()">+</button><br>
+    <button onclick="button4()">4</button>
+    <button onclick="button5()">5</button>  
+    <button onclick="button6()">6</button>
+    <button onclick="buttonSub()">-</button><br>
+    <button onclick="button7()">7</button>
+    <button onclick="button8()">8</button>
+    <button onclick="button9()">9</button>
```

Comment:
you are creating events for every button which is hardcoded in HTML, Maybe you can try to use some logic and change this code into javascript and perform event registering there.

Reply:
Yes, to avoid this I have created another file 11-exercise where I have coded separately in javascript.Only think i have not done CSS styling to it.rest all logic implies to it.